src/filters/index.js

```diff
@@ -63,20 +63,23 @@ export default {
   /**
    * 12345 => $12,345.00
    *
-   * @param {String} sign
+   * @param {String} currency
+   * @param {Number} places
    */
 
-  currency (value, currency) {
+  currency (value, currency, places) {
     value = parseFloat(value)
     if (!isFinite(value) || (!value && value !== 0)) return ''
     currency = currency != null ? currency : '$'
-    var stringified = Math.abs(value).toFixed(2)
-    var _int = stringified.slice(0, -3)
-    var i = _int.length % 3
+    places = places != null ? places : 2
```

Comment:
Instead of having a default value of 2, I think it's better to not use `toFixed` is no value is passed

What do you think?